suites/squid/rados/tier-2_rados_test-bugfixes.yaml

```diff
@@ -261,6 +261,8 @@ tests:
 
   - test:
       name: Verify that there are no CEPHADM_STRAY_DAEMON warning while replacing the osd
-      module: test_No_Cephadm_Stray_Daemon.py
```

Comment:
From @pdhiran : Delete the redundant test case module - test_No_Cephadm_Stray_Daemon.py